Inter-views/Inter-views
# Dependencies initialization

## **Types of Changes**


## **Purpose**
- initialize webpack, eslint, babel, and dependencies

## **Approach**
- articulate config files and package.json

## **How to Test**
- npm run build // should create dist folder with bundled front-end

- npm start // should spin up server on port 3000

-npm run dev // should enable dev server with HMR ( requires page-reload to enable HMT )
